addon/maptools/rectangletool.js

```diff
@@ -3,31 +3,35 @@ import MapTool from 'ember-flexberry-gis/maptools/maptool';
 export default MapTool.extend({
 
   _layer: null,
+  _measure: null,
 
-  rectangleStarted({ layer }) {
```

Comment:
Why zoom, zoom out, identify are subclass of  rectangletool? What properties and method (except for rectangleStarted) they inherit?
I think we should discuss this and related (use disableEdit()) questions on Monday...